content/en/dei/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Diversity, Equity, and Inclusion"
+date: 2021-09-17
+draft: false
+---
+
+Healthy communities are built when everyone's voice is heard,
+when their perspective is valued, and when their work is recognized.
```

Comment:
I think the sentiment here is spot on, but this reads strangely to me. I don't really have any concrete ideas how to prove the wording, but maybe something like:

```suggestion
Healthy communities are built on the foundations of equitable access and inclusivity.
A community can only flourish when everyone's voice is heard,
when their perspective is valued, and when their work is recognized.
```

Reply:
I think is that to me it reads a little passive? Like the dominant majority needs to listen, recognize, value - which is true, but is also missing something like "empower folks to act", "take action to correct injustice" - dunno, but something more active that better centers the folks doing the work.